con-base/src/main/java/ba/codecentric/base/service/ImageServiceImpl.java

```diff
@@ -27,6 +30,7 @@ public String saveImage(InputStream image, String imageName) throws IOException
             newImageName = getUniqueImageName(imageName);
             Path filePath = Paths.get(directoryPath + newImageName);
             Files.copy(image, filePath);
+            log.info("Save image: " + imageName);
```

Comment:
With log you should say what happened. So maybe it would be better "Image " + imageName +  " saved". Same for all other log messages.